math-doc/math/scribblings/math-matrix.scrbl

```diff
@@ -409,6 +409,13 @@ where each entry in @racket[M] is multiplied with @racket[z].
                  (matrix-scale (matrix [[1 2] [3 4]]) 2)]
 }
 
+@defproc[(matrix-kronecker [M (Matrix Number)] [N (Matrix Number)] ...) (Matrix Number)]{
+Computes the Kronecker product.
+@examples[#:eval untyped-eval
+                 (matrix-kronecker (matrix [[1 2] [3 4] [5 6]])
+                                   (matrix [[7 8] [9 10]]))]
+}
+
```

Comment:
This should have a history note.

Reply:
Should I bump the version number in `info.rkt`? It's been `1.1` for a while now.

Reply:
That's up to you, but it seems reasonable.

Reply:
Just saw https://github.com/racket/math/commit/17debaefd291165030addd6b2de686828829868c. So I think bumping the version number makes sense.